deploy/051_deploy_worldchain_spokepool.ts

```diff
@@ -2,16 +2,26 @@ import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { deployNewProxy, getSpokePoolDeploymentInfo } from "../utils/utils.hre";
 import { FILL_DEADLINE_BUFFER, WETH, QUOTE_TIME_BUFFER, ZERO_ADDRESS, USDCe } from "./consts";
+import { toWei } from "../utils/utils";
+import { CHAIN_IDs } from "@across-protocol/constants";
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { hubPool, spokeChainId } = await getSpokePoolDeploymentInfo(hre);
+  const { hubPool, spokeChainId, hubChainId } = await getSpokePoolDeploymentInfo(hre);
 
   const initArgs = [
     1,
     // Set hub pool as cross domain admin since it delegatecalls the Adapter logic.
     hubPool.address,
     hubPool.address,
   ];
+
+  // Set the Hyperlane xERC20 destination domain based on the chain
+  // https://github.com/hyperlane-xyz/hyperlane-registry/tree/main/chains
+  const hypXERC20DstDomain = hubChainId == CHAIN_IDs.MAINNET ? 1 : 11155111;
```

Comment:
I just realized how often you're repeating this entire line. I recommend exposing a public `getHypXERC20DstDomain()` utility function, and also exposing a constants library (perhaps in deploy/consts) for the Hyperlane Destination domains.

Same suggestions for OFT's of course